modules/kafo_configure/manifests/init.pp

```diff
@@ -4,14 +4,7 @@
 #   $modulepath/config/answers.yaml
 #   /etc/kafo-configure/answers.yaml
 #
-# @param add_progress
-#   Whether to add a progress bar. Only works on Puppet < 6.
 class kafo_configure(
-  Boolean $add_progress = $::kafo_add_progress,
```

Comment:
If you're dropping this, please also drop all code for it:
https://github.com/theforeman/kafo/blob/a45b1ebedda66cdf460c3fe5bfbb3696f2bd198f/lib/kafo/puppet_command.rb#L76
https://github.com/theforeman/kafo/blob/a45b1ebedda66cdf460c3fe5bfbb3696f2bd198f/lib/kafo/puppet_command.rb#L82-L84
(and the tests)